tests/cross/src/test/scala/tests/pc/CompletionArgSuite.scala

```diff
@@ -202,6 +202,17 @@ class CompletionArgSuite extends BaseCompletionSuite {
     ""
   )
 
+  check(
+    "arg14",
+    s"""|object Main {
+        |  curry(1)(bana@@)
+        |  def curry(a: Int)(banana: Int): Int = ???
+        |}
+        |""".stripMargin,
+    """|banana = : Int
```

Comment:
This test case doesn’t stress the case where there is a conflicting variable in scope, do you think it’s worth adding another “bananaFoo” val above to test that the default completions is still unchanged?

Reply:
Changed the test case to show the actual situation from the issue.